torch/utils/hipify/version.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.0'
+__version__ = '2.0.0'
```

Comment:
What's the impact of this?

Reply:
The version has never been updated even as we added new cuda-to-hip mappings.  The change is needed now because we're making BC-breaking changes to torch's hipify.  We have a related project hipify_torch that C++ projects can use --- it was an attempt years ago to make hipify into a standalone tool that could be a torch thirdparty submodule but we decided it would have negatively impacted developer experience for torch by making it a submodule.  So we have both the hipify that's inside torch and the hipify_torch c++ cmake helper project.  The hipify_torch project might query the torch.utils.hipify.__version__ so it can conditionally perform the corresponding hipify steps, v1 or v2.